javascripts/app.js

```diff
@@ -30,6 +29,62 @@ console.log("spell: ", spell.toString());
    */
   $("#player-setup").show();
 
+  // Display Name
+  $("#nameChoice").on("click", function() {
+    warrior.name = $("#player-name").val();
+    $("#battleName").append(warrior.name);
+  });
+
+  $(".classChoice").on("click", function() {
+    var humanClass = this.id
+
+    switch(humanClass) {
+      case "warrior":
+        warrior.setClass(humanClass);
+        break;
+      case "valkyrie":  
+        warrior.setClass(humanClass);
+        break;
+      case "berserker":  
+        warrior.setClass(humanClass);
+        break;
+      case "monk":  
+        warrior.setClass(humanClass);
+        break;
+      case "wizard":  
+        warrior.setClass(humanClass);
+        break;
+      case "sorcerer":  
+        warrior.setClass(humanClass);
+        break;
+      case "conjurer":  
+        warrior.setClass(humanClass);
+        break;
+      case "surprise":
+        warrior.generateClass();
+        break;
+
+    }
+      console.log("Player", warrior)      
+  })
+
+  $(".weaponClass").on("click", function() {
+    var pickWeapon = this.id
+
+    switch(pickWeapon) {
+      case "dagger":
```

Comment:
Can you think of a way to eliminate the need for this switch statement entirely? Hint, square bracket notation for accessing object keys (e.g. `Object[key]()`)